interfaces/builtin/registry_test.go

```diff
@@ -118,12 +118,12 @@ func (s *registrySuite) TestRegistrySanitizePlug(c *C) {
 		{
 			account: "my-acc",
 			view:    "_foo/bar",
-			err:     `invalid registry name: _foo`,
+			err:     `invalid registry name: _foo does not match ^[a-z0-9](?:-?[a-z0-9])*$`,
```

Comment:
I think I'd personally prefer to use `%q` above so that the regexp in the error message is surrounded by `'` characters.

Reply:
Actually, `%q` would quote the characters unpleasantly, I think adding `'''` or `'`'` (backtick) characters around the `%s` in the error message is what I'd prefer.

Reply:
It's now wrapped in single quotes. Thanks